pkg/kubelet/server/remotecommand/httpstream.go

```diff
@@ -176,6 +189,56 @@ type protocolHandler interface {
 	waitForStreams(streams <-chan streamAndReply, expectedStreams int, expired <-chan time.Time) (*context, error)
 }
 
+// v3ProtocolHandler implements the V3 protocol version for streaming command execution.
+type v3ProtocolHandler struct{}
```

Comment:
Is the problem here that we have to wait for the resize stream in order to know?  Do we really need a new protocol?